<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_DAG_File"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionEdit_Controls"/>
    <addaction name="actionShow_Vertex_Normals"/>
    <addaction name="actionShow_Geometry"/>
    <addaction name="actionShow_Bones"/>
    <addaction name="actionShow_Cloth_Bones"/>
    <addaction name="actionShow_Bone_Names"/>
    <addaction name="actionShow_Ground_Plane"/>
    <addaction name="actionBone_Hierarchy"/>
    <addaction name="actionAnimations"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
  </widget>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Open_DAG_File">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEdit_Controls">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Controls</string>
   </property>
  </action>
  <action name="actionShow_Vertex_Normals">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Vertex Normals</string>
   </property>
  </action>
  <action name="actionShow_Geometry">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Geometry</string>
   </property>
  </action>
  <action name="actionShow_Bones">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Bones</string>
   </property>
  </action>
  <action name="actionShow_Cloth_Bones">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Cloth Bones</string>
   </property>
  </action>
  <action name="actionShow_Bone_Names">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Bone Names</string>
   </property>
  </action>
  <action name="actionShow_Ground_Plane">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Ground Plane</string>
   </property>
  </action>
  <action name="actionBone_Hierarchy">
   <property name="text">
    <string>Bone Hierarchy</string>
   </property>
  </action>
  <action name="actionAnimations">
   <property name="text">
    <string>Animations</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
